versions.tf

```diff
@@ -1,4 +1,33 @@
-
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```

Comment:
@danielhoherd I don't think this will be backwards compatible. My thought was to note on our site or in a README that support for 0.13 begins in module version X. What do you think is the best way to handle this?

Reply:
That's fine. We should probably do a major version bump for that since it is essentially a functionality change. IE: it will break existing installations.